cmd/commandfuncs.go

```diff
@@ -64,6 +65,14 @@ func cmdStart(fl Flags) (int, error) {
 	// ensure it's the process we're expecting - we can be
 	// sure by giving it some random bytes and having it echo
 	// them back to us)
+
+	// load all additional envs as soon as possible
+	if startCmdEnvfileFlag != "" {
+		if err := loadEnvFromFile(startCmdEnvfileFlag); err != nil {
+			return caddy.ExitCodeFailedStartup,
+				fmt.Errorf("loading additional environment variables: %v", err)
+		}
+	}
```

Comment:
Instead of doing this, I would just pass the flag through to `run` like the other flags below. (Also, this section is in the way, between the comment before it which relates to the next code after it.)

Reply:
thanks ,i've fixed it~